### PR TITLE
Simplify wording for two_sided_door_search

### PR DESCRIPTION
**Pull request by henriquegemignani**
image

### CODE REVIEW

**Review by dyceron — APPROVED** [review]

**Comment by henriquegemignani** [comment]
image
Moved setting to the Door Lock tab

**Comment by henriquegemignani** [comment]
... just realized I could put in the same group

**Comment by henriquegemignani** [comment]
image

**Comment by Miepee** [comment]
can you please only make it show while the dropdown is on "doors"?

**Comment by henriquegemignani** [comment]
> can you please only make it show while the dropdown is on "doors"?

I thought about it, but it's rather complicated because it becomes two different things trying to hide it. Don't wanna figure that out

**Comment by Miepee** [comment]
ah right, since experimental stuff is handled slightly differently

File: randovania/gui/ui_files/preset_dock_rando.ui [hunk 1]
```diff
@@ -89,17 +110,20 @@
              </property>
             </widget>
            </item>
-           <item row="5" column="0" colspan="2">
-            <widget class="QLabel" name="multiworld_label">
+           <item row="9" column="0" colspan="2">
+            <widget class="QLabel" name="two_sided_door_search_label">
              <property name="text">
-              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The selected type is &lt;span style=&quot; font-weight:700;&quot;&gt;not&lt;/span&gt; compatible with multiworld sessions.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When using type Doors, enables the use of a revised algorithm for deciding what doors can be.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (Miepee) [inline randovania/gui/ui_files/preset_dock_rando.ui:116]: i dont like how this can be misinterpreted, but thats already the case for other places too :(

**Review by Miepee — APPROVED** [review]

**Pull request merged**